package.json

```diff
@@ -16,7 +16,7 @@
     "lint:docs": "markdownlint '**/*.mdx'",
     "prepublishOnly": "npm run build",
     "pretest": "npm run clean",
-    "release": "semantic-release --no-ci --extends ./release-local.config.cjs",
+    "release": "bash ./scripts/release",
```

Comment:
because you have the shebang `#!/usr/bin/env bash` you can just execute this without 'bash'

`"release": "./scripts/release.sh",`